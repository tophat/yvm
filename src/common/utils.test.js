const { getExtractionPath } = require('./utils')

describe(`utils`, () => {
    it(`getExtractionPath`, () => {
        const path = getExtractionPath()
        expect(path).toMatchSnapshot()
    })
})
