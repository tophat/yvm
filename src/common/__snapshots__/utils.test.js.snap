// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils getExtractionPath 1`] = `"/Users/jakebolam/.yvm/versions/vundefined"`;
